docs/dev/docker-hub.md

```diff
@@ -157,13 +157,15 @@ The ScyllaDB image supports many command line options that are passed to the `do
 
 #### `--seeds SEEDS`
 
-The `-seeds` command line option configures ScyllaDB's seed nodes.
-If no `--seeds` option is specified, ScyllaDB uses its own IP address as the seed.
+The `--seeds` command line option configures ScyllaDB's seed node.
+To start a single-node ScyllaDB cluster, pass the localhost address (127.0.0.1) as the `--seeds` argument.
+If no `--seeds` option is specified and both the listen address and broadcast address are equal to the localhost address
+or the `--developer-mode=1` option is used, ScyllaDB automatically uses the localhost address as the seed.
 
-For example, to configure ScyllaDB to run with two seed nodes `192.168.0.100` and `192.168.0.200`:
+For example, to configure ScyllaDB to run with seed node `192.168.0.100`:
 
 ```console
-$ docker run --name some-scylla -d scylladb/scylla --seeds 192.168.0.100,192.168.0.200
+$ docker run --name some-scylla -d scylladb/scylla --seeds 192.168.0.100
```

Comment:
@fruch do we have dtests for docker?
@ShlomiBalalis do we have SCT tests with docker?

Reply:
It's a bit embarrassing, but I've never ran scylla via docker, so I'm not sure how to test it.

Reply:
- we don't have dtest for the docker image
- we do have test in SCT to cover the basic function of a single docker instance (artifact-docker-test)

SCT is capable of creating a cluster of docker instances, but don't have anything like that that runs regularly (at least not yet) 

but in this case, I think it would be best just follow our instructions, and make sure they are working. (should be copy paste of just several commands)

Reply:
I'm not sure why are we removing the multiple seed examples from the docs ?

Reply:
> I'm not sure why are we removing the multiple seed examples from the docs ?

https://github.com/scylladb/scylladb/pull/15838#discussion_r1405821842